Tasks/DeployAzureResourceGroup/AzureUtilityLTE9.8.ps1

```diff
@@ -37,15 +37,18 @@ function Deploy-AzureResourceGroup
           [string]$overrideParameters)
 
     Switch-AzureMode AzureResourceManager
-    Write-Host "[Azure Resource Manager]Creating resource group deployment with name $resourceGroupName"
+
+    $deploymentName = [System.IO.Path]::GetFileNameWithoutExtension($csmFile) + '-' + ((Get-Date).ToUniversalTime()).ToString('yyyyMMdd-HHmm')
+
+    Write-Host "[Azure Resource Manager]Creating resource group deployment with name $deploymentName"
```

Comment:
The resource group name is displayed earlier (when it is created). This log is for describing the name of the deployment that is about to be created. Make sense?